### ISSUE TITLE
Error when loading OsirisRex profile

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#2177]
With a fresh `sync` folder, getting:

Lua: When initializing "A:\\openspace\\OpenSpace\\data\\assets\\scene\\solarsystem\\missions\\osirisrex\\model.asset": ...\assets\scene\solarsystem\missions\osirisrex\model.asset:175: Error loading scene graph node: (Spice) Kernel 'A:\openspace\sync\http\osirisrex_kernels\3\orx_r_160909_160910_v01.bc' does not exist

Which causes the profile not to load

**Comment by GPayne** [comment]
All of the v3 kernel files listed in the Osirix-REx kernels [asset](https://github.com/OpenSpace/OpenSpace/blob/master/data/assets/scene/solarsystem/missions/osirisrex/kernels.asset) seem to be accessible from any of our data servers except **orx_r_160909_160910_v01.bc**, which is only be available from the Utah server (didn't get copied over originally?). Copying it from [here](http://openspace.sci.utah.edu/files/solarsystem/missions/osirisrex/kernels/v1_v3/background/ck/orx_r_160909_160910_v01.bc) to the other data server(s) should fix the problem.

**Comment by alexanderbock** [comment]
Yes, it was an oversight to copy the files,.  Updated the kernel version to 4 which now includes the missing kernel
